config/webpack.config.js

```diff
@@ -81,7 +95,6 @@ if (production) {
     port: devServerPort,
     headers: { 'Access-Control-Allow-Origin': '*' }
   };
-  config.output.publicPath = '//0.0.0.0:' + devServerPort + '/webpack/';
```

Comment:
TBH I'm not sure why it was needed in the first place. I am removing it because it makes urls from css turn into absolute urls instead of relative ones, but that isn't needed because relative urls will work for both when the webpack dev server is enabled and disabled. (it would be relative to the relevant server - either rails or webpack)